packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -245,7 +245,9 @@ class Hyperlink extends React.Component {
           ref={this.linkRef}
         >
           <span className={cx('button-inner')}>
-            {text || children}
+            <span className={cx('inner-text')}>
```

Comment:
Shouldn't we be able to accomplish this with one span?

Reply:
As per the requirements, we need to make only the text have an ellipsis while the icon stays at the same level. So we need to use 2 spans for this. Otherwise it will start wrapping around if we try to implement the ellipses without having an additional inner span.